src/components/atoms/shared/Button/index.jsx

```diff
@@ -3,20 +3,20 @@ import styled from 'styled-components';
 import ButtonText from '../Typography/ButtonText';
 
 const stylesMap = {
-	primary: {
```

Comment:
Don't name the types according to functionality. Instead, try to name it according to style. This will increase the ease of understanding.

Also change the color, bgColor as per Figma. I had just added them as placeholder values.